plugins/arangodbendorsements/main.go

```diff
@@ -610,45 +449,25 @@ func (e *ArangoEndorsementStore) GetAltSoftwareComponents(args common.QueryArgs)
 // GetSwIDTagsForQueryFromDB is the most generic function that is been used to fetch
 // SWID's based on given query as an argument to the function.
 // Please note returned swid list contains only naked nodes, it does not contain any links/relations among swid.
-func (e *ArangoEndorsementStore) GetSwIDTagsForQueryFromDB(query string) ([]SoftwareIdentity, error) {
+func (e *EndorsementStore) GetSwIDTagsForQueryFromDB(query string) ([]SoftwareIdentity, error) {
+	var swidList []SoftwareIdentity
+	var Swdocwrap SoftwareIdentityWrapper
```

Comment:
I nearly dislocated my jaw trying to spell this variable's name! :-)

What is it supposed to mean?

Reply:
Sorry, this is simply software document wrapper acronymed as swdcowrap :)